cypress/integration/Add_profile_spec.js

```diff
@@ -0,0 +1,83 @@
+describe("add_notes_spec", () => {
+    it("Login", () => {
+        cy.fixture("profile").then((user) => {
+            cy.login({email: user.admin.email, password: user.admin.password});
+        });
+        cy.visit("http://localhost:3000/accounts");
+    });
+    it("Will go into the accounts tab and find the first instructor", () => {
+        cy.contains('INSTRUCTORS').click();
+        cy.get('a[href*="/accounts/instructor/4"]').click();
+        cy.contains("Notes").click();
+    });
+    it("Will create a note with a subject and body", () => {
+        cy.get("[data-cy=add-note]").click();
+        cy.contains("Save").should("be.disabled");
+        cy.get("[data-cy=textfield]").type("Out of office for today");
+        cy.get("[data-cy=note-body]").type("Daniel has some things to take care of and will not be coming in today");
+        cy.contains("Save").click();
+        cy.get("[data-cy=note-header]").contains("Out of office for today");
+        cy.get("[data-cy=body]").contains("Daniel has some things to take care of and will not be coming in today");
+    });
+    it("Will create a note with a notification with a subject and body", () => {
+        cy.get("[data-cy=add-note]").click();
+        cy.contains("Save").should("be.disabled");
+        cy.get("[data-cy=textfield]").type("Out of office for a couple days");
+        cy.get("[data-cy=note-body]").type("Daniel is sick and will not be in class for the next few days");
+        cy.get("[data-cy=dialog-notification]").click();
+        cy.contains("Save").click();
+        cy.get("[data-cy=note-header]").contains("Out of office for a couple days");
+        cy.get("[data-cy=body]").contains("Daniel is sick and will not be in class for the next few days");
+        cy.get("[data-cy=edit-btn]").last().click();
+        cy.get("[data-cy=dialog-notification]").should('have.css', 'color', 'rgb(255, 0, 0)');
+        cy.contains("Cancel").click();
+    });
+    it("Will display the number of important notes on the notes badge", () => {
+        cy.get("[data-cy=note-tab]").last().should('have.text', '1 Notes');
+    })
+    it("Will delete all notes", () => {
+        cy.get("[data-cy=delete-btn]").last().click();
+        cy.get("[data-cy=delete-confirm]").click();
+        cy.get("[data-cy=delete-btn]").first().click()
+        cy.get("[data-cy=delete-confirm]").click();
+    });
+    it("Will go into the registrations after login, click on AP Computer Science, and click on notes", () => {
```

Comment:
If this is in the same test, why do we need to login again?

Reply:
when using cy.visit(), it reloads the page so if we don't login the test does not run.